tests/testthat/test-vars-rename.R

```diff
@@ -26,8 +25,7 @@ test_that("when .strict = FALSE, vars_rename always succeeds", {
 
   expect_error(
     vars_rename(c("a", "b"), d = "e", f = "g", .strict = TRUE),
-    "Unknown columns `e` and `g`",
-    fixed = TRUE
+    class = "tidyselect_error_index_oob_names"
```

Comment:
Do you think it's worth testing for that the error contains e and g?

Reply:
It now only contains `e`. We'll get a more informative error when we merge rename and select implementations. Except for character inputs since those need to be validated early.

I have added a `verify_output()` to track these messages.